Kwenta/kwenta
# Resolve rate from coinGecko if exchange rate is not found

## Description

- Resolve currency rate from `coinGecko` if its key is not found in `exchangeRates`.
- Fix a bug when calculating exchange rate that base and quote currency key are passed to function `newGetExchangeRatesForCurrencies` with wrong order: https://github.com/Kwenta/kwenta/blob/51068f9900e9f597adc85aabd9825ea014179a5c/hooks/useExchange.ts#L278
### **After the fix, we should use `rate` but  `inverseRate` to render title in exchange page**

## Related issue
 
[Issue](https://github.com/Kwenta/kwenta/issues/1273)

## Motivation and Context

The title in exchange won't show correct rate if one or both currencies are not found in exchange rates results.

## How Has This Been Tested?



- Go to `Exchange` view
- Select `sINR` and `DAI`
- Tab title in browser should show correct exchange rate
- Switch currency to see title change

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/3646680/185224681-23751f8c-bee9-4f39-a9bf-6f125acdfe01.png)
![image](https://user-images.githubusercontent.com/3646680/185224736-06ea5af1-caa6-4578-8687-a0e136148434.png)
